virttest/env_process.py

```diff
@@ -1240,6 +1240,22 @@ def preprocess(test, params, env):
             test.cancel("Got host qemu version:%s, which is not in %s" %
                         (host_qemu, required_qemu))
 
+    # Checking required virtio driver version, if not satisfied, cancel test
+    if params.get("required_virtio") and params.get("cdrom_virtio"):
```

Comment:
Oh here is the limitation, please ignore below comment then. But `required_virtio` is confusing, has to be `required_virtio_win`, as this refers to package name, not same to `cdrom`.